src/sentry/static/sentry/app/components/userBadge.jsx

```diff
@@ -9,7 +9,7 @@ const UserBadge = ({user, orgId}) => {
       <Avatar user={user} size={80} />
       <h5>
         <Link to={`/settings/organization/${orgId}/members/${user.id}`}>
-          {user.email}
+          {user.name || user.email}
```

Comment:
huh, the email works as a route?

Reply:
what do you mean? the id is used for the url, the email is for the display